go-controller/pkg/node/helper_linux.go

```diff
@@ -54,3 +56,22 @@ func getIntfName(gatewayIntf string) (string, error) {
 	}
 	return intfName, nil
 }
+
+func deleteConntrack(ip string) {
```

Comment:
@girishmg fyi relates to your patch for your host <--> host back service patch you are adding. I think UDP traffic would also have the same issue across that service NAT